src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbByTable.java

```diff
@@ -230,7 +230,7 @@ public static MtAmazonDynamoDbBuilder builder() {
         return new MtAmazonDynamoDbBuilder();
     }
 
-    public static class MtAmazonDynamoDbBuilder implements TableBuilder {
+    public static class MtAmazonDynamoDbBuilder {
```

Comment:
Hi @degaydorus, seems like since I removed the other `SharedTable`-based strategies that this interface is no longer needed.  Can you confirm?

Reply:
The `TableBuilder` is currently used by clients of mt-dynamo. For example a client might use SharedTableBuilder and MtAmazonDynamoDbBuilder.  The `TableBuilder` interface was added with the intention for common table methods to be included there (not just billing mode), so client's could take advantage (minimize repeat code by table type).  As long as we have > 1 TableBuilder type with common methods, it's useful to have.

Reply:
Yeah, this falls into the bucket of stuff(mostly in builders) that is useful for clients but has no tests that actually exercise its use.  Reverted in 3fe9c5150c90f4af1f851cfa81ef95c8b88e8e2c.

Reply:
@msgroi, I added these tests mt-dynamo side to demonstrate TableBuilder use on client side: https://github.com/salesforce/mt-dynamo/pull/350.  Of course the weak cast is not needed, but it's only for test purposes.